### PR TITLE
Create projects page and aboutme page

### PR DESCRIPTION
**Pull request by ram1117**
1.Created and styled projects section. Used grid layout to align project tiles and flexbox for each project tile content
2.Created and styled about me section. Used grid layout for about me and skill sets segment. ul and li used for skillset content
3.Optimized style.css to reuse styling for buttons, section headers and text content
4.Added button hover, button click effects according to the project design template
5.Updated readme.md with new features added to the project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,23 +16,150 @@
...
+        <!-- porjects information section -->
+        <section class="projects">
+            <h2 class="section-header">Projects</h2>
+            <div class="projects-container">
+                <!-- individual project tile -->
+                <div class="project-tile">
+                    <img src="res-images/Rectangle 21.png" alt="open laptop top view" class="project-thumbnail">
+                    <div class="project-details">
+                        <h3 class="project-title">Project 1</h3>
+                        <div class="tech-item-container">
```
Comment (nedjwab) [inline index.html]: [ ] The list of project technologies should be a list `ul` and each of them a `li`.kindly fix that 🤓

File: index.html [hunk 2]
```diff
@@ -16,23 +16,150 @@
...
+                        <button class="button-generic button-project">See this project</button>
+                    </div>
+                </div>
+                <!-- individual project tile -->
+                <div class="project-tile">
+                    <img src="res-images/Rectangle 22.png" alt="open laptop and a mobile" class="project-thumbnail">
+                    <div class="project-details">
+                        <h3 class="project-title">Project 2</h3>
+                        <div class="tech-item-container">
+                            <div class="tech-item">HTML/CSS</div>
```
Comment (nedjwab) [inline index.html]: [ ] The list of project technologies should be a list `ul` and each of them a `li`.kindly fix that 🤓

File: index.html [hunk 3]
```diff
@@ -16,23 +16,150 @@
...
+                        <button class="button-generic button-project">See this project</button>
+                    </div>
+                </div>
+                <!-- individual project tile -->
+                <div class="project-tile">
+                    <img src="res-images/Rectangle 26.png" alt="open laptop tilted to right" class="project-thumbnail">
+                    <div class="project-details">
+                        <h3 class="project-title">Project 3</h3>
+                        <div class="tech-item-container">
+                            <div class="tech-item">HTML/CSS</div>
```
Comment (nedjwab) [inline index.html]: [ ] The list of project technologies should be a list `ul` and each of them a `li`.kindly fix that 🤓

File: index.html [hunk 4]
```diff
@@ -16,23 +16,150 @@
...
+                        <button class="button-generic button-project">See this project</button>
+                    </div>
+                </div>
+                <!-- individual project tile -->
+                <div class="project-tile">
+                    <img src="res-images/Rectangle 46.png" alt="open laptop front view 2" class="project-thumbnail">
+                    <div class="project-details">
+                        <h3 class="project-title">Project 5</h3>
+                        <div class="tech-item-container">
+                            <div class="tech-item">HTML/CSS</div>
```
Comment (nedjwab) [inline index.html]: [ ] The list of project technologies should be a list `ul` and each of them a `li`.kindly fix that 🤓

**Review by nedjwab — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @ram1117 

Nice job, you are almost there.



## (Highlights) Good Points: :
- Good implementation of some of the project requirements  ✔
- GitHub flow was correctly used ✔.

## (Changes Required) Aspects to improve: ♻️
You did great implementing some of the project requirements, but you still have some issues: 
 check the comments below 👇

 
## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by ram1117** [comment]
1.Restructured project technologies inside project tile from flexbox div to ul-li (code review 1)
2. Made some alignment changes to unordered lists in skill set tiles.
3. Rectified stylint errors in style.css

**Review by victorokoroji — APPROVED** [review]
Hi @ram1117, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

- Your readme seems to be missing the acknowledgment section. Kindly add it to make your readme more professional 👍🏽 

- I will strongly recommend you always write descriptive commit messages in imperative format and always capitalize the first letter of your commit messages 👍🏽 
e.g Instead of writing `rectified stylelint error`, you can write `Rectify stylelint error` which is imperative 😄 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**